homeassistant/components/vallox/config_flow.py

```diff
@@ -88,24 +86,24 @@ async def async_step_reconfigure(
         self, user_input: dict[str, Any] | None = None
     ) -> ConfigFlowResult:
         """Handle reconfiguration of the Vallox device host address."""
-        self._context_entry = self._get_reconfigure_entry()
         return await self.async_step_reconfigure_confirm()
```

Comment:
Any specific reason why you've removed this step on other integrations but left it here?

Reply:
I took it out when it was new (not in 2024.10)
I left it when it was old